packages/hooks/src/useIsMt5LoginListStatusPresent.ts

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useMT5AccountsList } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+// TODO: Remove this todo once 'open_order_position_status' flag is available in BE response and in type TSocketResponseData<"mt5_login_list">
+type DetailsOfEachMT5LoginId = ReturnType<typeof useStore>['client']['mt5_login_list'][number];
+type TStatus = keyof DetailsOfEachMT5LoginId;
```

Comment:
`DetailsOfEachMT5LoginId` type is taken from store, and its keys will be displayed to user as suggestions.
![Screenshot 2023-09-21 at 12 39 26 PM](https://github.com/shaheer-deriv/deriv-app/assets/122449658/d04272e1-3e5d-40cb-a02b-3132063dbd82)